### PR TITLE
SQuAD Demo

### PR DESCRIPTION
**Pull request by ksurya**
Closes https://github.com/gamma-lab/buffet/issues/42

### CODE REVIEW

**Comment by harrywang** [comment]
@shengoneconnect please finish the review this morning and get this merged.

**Comment by shengoneconnect** [comment]
1. readme: http://localhost:8881 instead of http://localhost:5000
2. please specify how to run the model locally (use qa_system_predict function from squad.py from my understanding) in addition to flask application

**Comment by ksurya** [comment]
Updated

File: squad_demo/squad.py [hunk 1]
```diff
@@ -0,0 +1,696 @@
...
+        for k, v in self.defaults.items():
+            setattr(self, k, v)
+        for k, v in kw.items():
+            setattr(self, k, v)
+
+
+def _load():
+    args = Arguments(
+        bert_model="bert-large-uncased",
+        output_model_file="bert_pytorch_model.bin",
```
Comment (leon0707) [inline squad_demo/squad.py]: should not be hardcoded

**Comment by leon0707** [comment]
related: https://github.com/gamma-lab/gamma_demos/issues/10

**Comment by leon0707** [comment]
bug https://github.com/gamma-lab/gamma_demos/issues/12

**Comment by ksurya** [comment]
Updated

**Comment by leon0707** [comment]
Will new line affect the result?
https://github.com/gamma-lab/gamma_demos/issues/3

seems like allennlp doesn't remove the new line. They can highlight the answer in the original paragraph. In the demo, the original input has to be modified to highlight the correct positions.

**Comment by ksurya** [comment]
is the issue still relevant?

**Pull request merged**